exonum/src/storage/indexes_metadata.rs

```diff
@@ -154,8 +161,35 @@ pub fn assert_index_type(name: &str, index_type: IndexType, is_family: bool, vie
     }
 }
 
+#[derive(Debug, Clone)]
+pub(crate) enum StorageVersion {
+    Supported(VersionValue),
+    Unsupported(VersionValue),
+    Unspecified,
+}
+
+impl StorageVersion {
+    pub(crate) fn current() -> VersionValue {
```

Comment:
Nit: I think it's unnecessary to use `pub(crate)` modifiers for methods in the type that is `pub(crate)` itself.